src/store/traction/ont/getters.js

```diff
@@ -1,6 +1,155 @@
+import { wellToIndex } from './wellHelpers'
+
+const mergeRepresentations = (parent, child, keyFunction = (id) => id) => {
+  return Object.values(parent).map((parentRecord) => {
+    return { ...child[keyFunction(parentRecord.id)], ...parentRecord }
+  })
+}
+
+const sortRequestByWellColumnIndex = (resources) => (a, b) =>
```

Comment:
Not sure what is going on here and it looks quite clever so definitely need some comments.